pkg/components/networking.go

```diff
@@ -63,6 +77,7 @@ func startOVNKubernetes(cfg *config.MicroshiftConfig, kubeconfigPath string) err
 		return err
 	}
 	extraParams := assets.RenderParams{
+		"MTU":            ovnConfig.MTU,
```

Comment:
What do you think about passing ovnConfig here so the template can access any future settings without us having to extend this parameter list?

Reply:
Good idea, will update to pass ovnConfig.

Reply:
done.